setup/azurecmdfiles.wxi

```diff
@@ -8,6 +8,9 @@
                         <Component Id="cmp0761C025FCF8EF027A5BC23C72777C52" Guid="*">
                             <File Id="fil80798000B169AC6D62DD8331A55CF6FF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\Azure.Storage\Azure.Storage.psd1" />
                         </Component>
+                        <Component Id="cmpCC0D4240D3640D2D425852E0F2A4D31E" Guid="*">
```

Comment:
No, I'm saying the script should only be copied into the folder you need it, not into 3 folders